tsconfig.es5.json

```diff
@@ -5,5 +5,6 @@
     "outDir": "dist/es5",
     "rootDir": "src",
     "target": "es5",
+    "tsBuildInfoFile": "node_modules/.cache/uniforms-semantic.es5.tsbuildinfo"
```

Comment:
Please change the chached filename to `uniforms-patternfly` instead of `uniforms-semantic`.

Reply:
This is actually meant to be that, it's, for some reason, needed to point to what the core uniforms lib did build.
At least, that's what I understood looking at the upstream lib...

Reply:
Ah no, I just noticed the mistake, I'm wrong. I'll fix it.